### ISSUE TITLE
Tests never end on Travis-CI

### ISSUE THREAD

**Issue by MoOx** [airportyh/testem#443]
Yeah sound weird to run testem on Travis, but acutally, it's better than nothing (I plan to run Firefox + phantomjs).
Here is a problematic build https://travis-ci.org/viareport/starterkit-react-cssnext-webpack/builds/43749854
I have noted that on my machine, CI mode is sometimes really slow to end and even some times I needed to kill using ctrl+c.
Any idea on what can create this issue?

**Comment by johanneswuerbach** [comment]
Which test framework are you using and are you sure that you are closing all async waits?

**Comment by MoOx** [comment]
tape & I don't have this issue locally. It was very a stupid & simple test like `t.pass(true)` just to setup testem before a real usage.

**Comment by MoOx** [comment]
You can see the code here https://github.com/viareport/starterkit-react-cssnext-webpack

**Comment by johanneswuerbach** [comment]
I tried to run your repo (git clone / npm install), but `npm run build-test` fails with the following:
```
npm run build-test

> frontend-hot-starterkit@0.0.0 prebuild-test /starterkit-react-cssnext-webpack
> npm run lint


> frontend-hot-starterkit@0.0.0 lint /starterkit-react-cssnext-webpack
> npm run eslint && npm run jscs


> frontend-hot-starterkit@0.0.0 eslint /starterkit-react-cssnext-webpack
> eslint . --ext=.js --ext=.jsx


scripts/build-test.js
  22:50  error  err is already declared in the upper scope  no-shadow

src/index.jsx
  11:3  error  Unexpected token <

web_modules/hello-universe/index.jsx
  11:9  error  Unexpected token (

web_modules/hello-world/index.jsx
  9:9  error  Unexpected token (

✖ 4 problems
```

**Comment by MoOx** [comment]
I will check that asap

**Comment by MoOx** [comment]
Don't want to open an issue for that, but I were having an issue locally this time. It seems that Opera (in my case) was not quit properly or something similar. When I remove it from my list, testem ends correctly otherwise, I get stuck testem ci.
No idea if this can be related, just want to keep a trace of this issue somewhere :)

**Comment by johanneswuerbach** [comment]
@MoOx please open a new issue, so we can handle and discuss it separately. Please provide also your node / testem / os and opera version.

**Comment by johanneswuerbach** [comment]
This should be finally fixed in `v0.9.1`.
